### PR TITLE
5291 firmware to replace the existing soarers controller firmware

### PR DESCRIPTION
**Pull request by listofoptions**
controller for the eponymous keyboard, def squash my history, im prone to many stupid hiccups

### CODE REVIEW

File: keyboards/converter/hp_46010a/keymaps/default/keymap.c [hunk 1]
```diff
@@ -34,7 +34,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
      * `-----------------------------------------------------------------------------------'    `-------------------'
      */
      
-    [0] = LAYOUT(
+    [0] = LAYOUT (
```
Comment (noroadsleft) [inline keyboards/converter/hp_46010a/keymaps/default/keymap.c]: This file should be reset.

`git checkout master -- keyboards/converter/hp_46010a/keymaps/default/keymap.c`

**Comment by noroadsleft** [comment]
I'd also suggest/recommend renaming the keyboard as `ibm_5291` rather than just `5291`.

**Comment by listofoptions** [comment]
renamed the keyboard too.

**Comment by listofoptions** [comment]
@noroadsleft seems y'all are a bit busy!

File: keyboards/converter/ibm_5291/rules.mk [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+SPLIT_KEYBOARD = no
+WAIT_FOR_USB = yes
+
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no    # Breathing sleep LED during USB suspend
+
+LAYOUTS_HAS_RGB = NO
+
+CUSTOM_MATRIX = yes
+SRC = matrix.c
```
Comment (drashna) [inline keyboards/converter/ibm_5291/rules.mk]: This should be `+=`.

```suggestion
SRC += matrix.c
```
But this should be fine.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**